### DISCUSSION TITLE
求教一下，本地存储的 上传按钮 在哪里？

### DISCUSSION THREAD

**Discussion by msxfXF** [AlistGo/alist#6368] [Q&A]
求教一下，本地存储的 上传按钮 在哪里？ 怎么上传文件？
![image](https://github.com/alist-org/alist/assets/38368353/3cd30aec-e963-43a5-9908-0824f072fbe2)

**Comment by xuanxuan1231** [comment]
![image](https://github.com/alist-org/alist/assets/106209242/2e856b81-cd6d-44a8-be0d-cf04f621da36)
右下角
